Dockerfile

```diff
@@ -59,7 +59,7 @@ FROM deps as dev
 # enforce perms
 USER root
 WORKDIR /home/unlock
-RUN chown -R node:node /home/unlock
+RUN chown -R node:node /home/unlock/.yarn/install-state.gz
```

Comment:
that chown step was actually taking several minutes (as it went thru all the node_modules folder) - while we only needed that file perm to be corrected